SparkPost/2web2ui
# EO-654 - Signals Injections Endpoint and Hover Component for Health Score Chart

Branching off new UI health score chart branch

### What Changed
- New signals injections endpoint action/reducer
- Health Score Injections Hover Component
- https://sparkpost.invisionapp.com/share/H8R0LV8E5NA#/352294734_Health_Dashboard

### How To Test
- [Point your local upstreams to staging](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams)
- Log into appteam or enable the `feature_signals_v2` UI option
- Navigate to the new Health Score page from the main side nav
- Verify the bar chart interaction (hovers) and updates injections
